tests/resources/test_common.py

```diff
@@ -320,9 +315,9 @@ def test_str(self):
         self.assertEqual(
             (
                 'PatientItemSet {\n'
-                '	patient1_id\n'
-                '	patient2_id\n'
-                '	patient3_id\n'
+                '	PatientItem(id="patient1_id")\n'
```

Comment:
How about also adding a `name` field for at least one of the patients to see the change in `repr`

Reply:
It will always include the name field if the object has a `name` attribute -- even if that attribute is empty.

Reply:
Cool nice update, I meant to say `name` attribute.